src/GooglePlay/Acknowledger.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace ReceiptValidator\GooglePlay;
+
+/**
+ * Class Acknowledger
+ * @package ReceiptValidator\GooglePlay
+ */
+class Acknowledger
+{
+    /**
+     * @var \Google_Service_AndroidPublisher
+     */
+    protected $_androidPublisherService = null;
+    /**
+     * @var string
+     */
+    protected $_package_name = null;
```

Comment:
null value is default and not required.

Reply:
It's the same logic as for the Validator class to preserve same usage and behavior

Reply:
I know. And it is nice to to a refactoring to be in the same code style.
BTW, it's not obligatory to fix that. Thanks.

Reply:
Ok I did it in the same time